packages/components/src/components/input-range/shadow.tsx

```diff
@@ -59,7 +64,7 @@ export class KolInputRange implements InputRangeAPI, FocusableElement {
 	private readonly catchInputNumberRef = (element?: HTMLInputElement) => {
 		if (element) {
 			this.refInputNumber = element;
-			if (!this._value && this.refInputNumber?.value) {
+			if (this._value !== undefined && typeof this.refInputNumber?.value === 'string') {
```

Comment:
```suggestion
			if (this._value === undefined && typeof this.refInputNumber?.value === 'string') {
```
Müsste es nicht so herum sein?